# python-fdb isn't in the Debian archive yet so default dialect for firebird://
# URLs is changed to obsolete kinterbasdb
Index: sqlalchemy-0.9.6/lib/sqlalchemy/dialects/firebird/__init__.py
===================================================================
--- sqlalchemy-0.9.6.orig/lib/sqlalchemy/dialects/firebird/__init__.py
+++ sqlalchemy-0.9.6/lib/sqlalchemy/dialects/firebird/__init__.py
@@ -6,7 +6,7 @@
 
 from sqlalchemy.dialects.firebird import base, kinterbasdb, fdb
 
-base.dialect = fdb.dialect
+base.dialect = kinterbasdb.dialect
 
 from sqlalchemy.dialects.firebird.base import \
     SMALLINT, BIGINT, FLOAT, FLOAT, DATE, TIME, \
